bats/fb-katello-client.bats

```diff
@@ -7,8 +7,16 @@ load os_helper
 load foreman_helper
 load fixtures/content
 
-@test "enable katello-agent" {
-  foreman-installer --foreman-proxy-content-enable-katello-agent true
+setup() {
+  KATELLO_VERSION=$(tKatelloVersion)
+}
+
+@test "enable remote execution (or katello-agent for Katello 4.9 and below)" {
```

Comment:
Another approach to make it clear is to have 2 "tests". One for enable katello-agent that's skipped on >= 4.10 and one for enabling REX that's skipped on < 4.10.

Or given how much of this file is conditional: write `fb-katello-client-rex.bats` and `fb-katello-client-katello-agent.bats` and handle it at a higher level. Simply don't execute it at all on >= 4.10.

Reply:
+1 from me to split file approach